FDBLibTLS/local.mk

```diff
@@ -1,6 +1,14 @@
+
+ifeq ($(PLATFORM),freebsd)
+FDBLibTLS_CFLAGS := -fPIC -I/usr/local/include -I$(BOOSTDIR)
+FDBLibTLS_LIBS := -lssl -lcrypto
```

Comment:
I'm surprised that this builds without -ltls, as it appears even the FreeBSD port for LibreSSL offers a libtls, and the TLS plugin implementation, I thought, relied on the friendlier TLS API/library.